dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1363,11 +1375,8 @@ restoreServer()
   # Store the current restored environment into a file that the JVM can read to restore the values.
   # The following are filtered out of the restore env because they could be different from the checkpoint
   # values.  The JVM only allows new values to be set into the restored environment:
-  # - JAVA_OPTIONS vars are not set the same on the restore side so we filter them out
-  # - SHLVL the shell level could be different
-  # - HOSTNAME the host name likely is always different for containers
-  # - _ holds the currently running process, here we do no need to have it set to `env`
-  env | grep -Ev "JAVA_OPTIONS=|SHLVL=|HOSTNAME=|_=|X_LOG_FILE=|X_LOG_DIR=" > ${SERVER_OUTPUT_DIR}/workarea/checkpoint/.env.properties
+  # - INVOKED - this gets set to the path used to launch the server script, we don't care if this changes
+  env | grep -Ev "INVOKED=" > ${SERVER_OUTPUT_DIR}/workarea/checkpoint/.env.properties
```

Comment:
Since this filters out any var ending in INVOKED should it be more specific like:
  env | grep -Ev "^INVOKED="

Reply:
Done